api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -9,7 +9,7 @@
 import org.pmiops.workbench.interceptors.CloudTaskInterceptor;
 import org.pmiops.workbench.interceptors.CorsInterceptor;
 import org.pmiops.workbench.interceptors.CronInterceptor;
-import org.pmiops.workbench.interceptors.ElapsedTimeDistributionInterceptor;
```

Comment:
questions is RequestTimeMetricInterceptor just a new rename for  ElapsedTimeDistributionInterceptor ? i cannot see it being removed as a file in this commit.

Reply:
Good catch. Somehow I wound up adding instead of renaming.